internal/graph/graph_test.go

```diff
@@ -312,3 +314,91 @@ func TestTrimTree(t *testing.T) {
 		}
 	}
 }
+
+func nodeTestProfile() *profile.Profile {
+	var mappings = []*profile.Mapping{
```

Comment:
I actually copied most of this from the driver testcase.

It would be really nice if we could write a text description of the profile in a string, that could be
automatically parsed into a profile.Profile. It would make tests like these more readable and easier to write.